tutorials/audio/audio_effects.rst

```diff
@@ -0,0 +1,192 @@
+.. _doc_audio_effects:
+
+Audio Effects
+=============
+
+Godot includes several audio effects that can be added to an audio bus to
+alter every sound file that goes through that bus.
+
+.. image:: img/audio_buses4.png
+
+Try them all out to get a sense of how they alter sound. Here follows a short
```

Comment:
Unrelated to this pull request, but we should probably have a demo in the demo projects repository for this :slightly_smiling_face: